specification/attributes/numeric_format.md

```diff
@@ -0,0 +1,91 @@
+# Numeric Format
+
+Columns that provide numeric values conforming to specified rules and formatting requirements ensure clarity, accuracy, and ease of interpretation for both humans and systems.
+
+All columns capturing a numeric value, defined in the FOCUS specification, MUST be a real number represented as an integer or a decimal value as defined in this section.
+
+## Attribute ID
+
+NumericFormat
+
+## Attribute name
+
+Numeric Format
+
+## Description
+
+Rules and formatting requirements for numeric columns appearing in billing data.
+
+## Requirements
+
+* Columns with a Numeric format MUST contain a single, numeric value.
+* Numeric values MUST be expressed as integer or decimal values. Fractional notation is NOT allowed.
+* Numerical values expressed with mathematical symbols, operators, or exponent values are not permitted.
+* Numeric values MUST NOT contain qualifiers or additional characters (e.g. currency symbols, units of measure, etc.).
+* Numeric values MUST NOT contain commas or punctuation marks except for a single decimal point if required to express a decimal value.
+* Numeric values MUST NOT include a character to represent sign for a positive value.  The only sign permitted is a negative sign (-) to indicate a negative value.
+* Numeric values MUST NOT be used to represent binary values (e.g. 0 or 1).
+
+## Numeric Precision
```

Comment:
Thoughts on this and the two sections below being a sub-heading and not the same level as the standard headers we use for the sepc content? For example we could park these as H3s after the normative content at the very top (before ## Attribute ID).

Reply:
ok, I rearranged content in the latest push to the branch.